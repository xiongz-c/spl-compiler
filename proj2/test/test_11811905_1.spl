struct myStruct{
    float m1;
}g1[2][3];

struct myStruct {

};

int test_func(){
    return 2.0;
}

int test_11811906_1(int v1, float v2) {
    int v1 = 2;
    g1[0][0].m1 = v2;

    return test_func();
}

