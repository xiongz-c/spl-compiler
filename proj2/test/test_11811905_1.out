Error type 15 at Line 5: redefine struct: myStruct
Error type 8 at Line 10: incompatiable return type
Error type 3 at Line 14: redefine variable: v1
